ideascube/mediacenter/forms.py

```diff
@@ -27,3 +30,30 @@ def save(self, commit=True):
         document.save()
         self.save_m2m()
         return document
+
+
+def CreateDocumentForm(path):
+    class NotFileDocumentForm(forms.ModelForm):
+
+        class Meta:
+            model = Document
+            exclude = ['original', 'preview']
+
+        original = forms.FilePathField(path=path)
+        preview = forms.FilePathField(path=path, required=False)
+
+        def save(self, commit=True):
+            document = super(NotFileDocumentForm, self).save(commit=False)
+            original = self.cleaned_data['original']
+            original = os.path.relpath(original, settings.MEDIA_ROOT)
```

Comment:
Why do you prefer storing the path relative to the media root?

I assume later, when opening the file, we have to `os.path.join(settings.MEDIA_ROOT, document.original)`, which defeats the purpose, doesn't it?